src/js/player.js

```diff
@@ -2256,6 +2256,11 @@ class Player extends Component {
           this.addClass('vjs-user-active');
           this.trigger('useractive');
         } else {
+          // if we are focused anywhere in the player, don't go inactive
+          if (player.el().contains(document.activeElement)) {
```

Comment:
maybe this should be `player.controlBar.el().contains(document.activeElement)`?
And then `<Esc>` would focus the player div?